minecraft_servers/gamerherz/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+  "server_name": "Gamerherz",
+  "nice_name": "Gamerherz.eu",
+  "direct_ip": "Gamerherz.eu",
+  "server_wildcards": [
+    "%.gamerherz.eu"
+  ],
+  "supported_languages": [
+     "de",
+     "en"
+  ],
+  "social": {
+    "web": "https://www.gamerherz.eu/",
```

Comment:
Returns a error 404

Reply:
The Website is in two or three days back online.